README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
+}, function(err, result) {
+  // result = {
+  //   transcript: 'how old is the Brooklyn Bridge',
+  //   confidence: 0.97234234234
+  // ]
+});
+```
+
+```js
+// Detect the speech in an audio file stream.
+var initialRequest = {
+  encoding: 'LINEAR16',
+  sampleRate: 16000,
+  continuous: true
+};
+
+var stream = speech.recognize(initialRequest);
+  .on('data', function (recognizeResponse) {
+    console.log(recognizeResponse);
+  });
+
+stream.write({
+  audioRequest: {
```

Comment:
I think I messed up on this method. I used the gcloud-node's Vision API implementation as an example, where it has the raw `annotate` method where gcloud-node doesn't do anything for you, and then the `detect` method that provides a higher level of abstraction. 

I'm thinking I'll renamed the current `detect` method to `nonStreamingDetect`, as it is the abstracted version of the grpc `nonStreamingRecognize` method. Then I'll fix the `recognize` method to be totally raw, then add a new `detect` method which is an abstracted version of the `recognize` method. So:

```js
var stream = speech.recognize(...) // you provide raw `RecognizeRequest` objects
var stream2 = speech.detect(...) // easier to work with, just provide the `InitialRequest` and audio chunks
speech.nonStreamingRecognize(...) // you provide a raw `RecognizeRequest` object
speech.nonStreamingDetect(...) // easier to work with, just provide a filename/uri and the `InitialRequest`
```